data-pipeline/src/stats_exporter.rs

```diff
@@ -82,30 +81,24 @@ impl StatsExporter {
         }
         let body = rmp_serde::encode::to_vec_named(&payload)?;
 
-        let headers: HashMap<&'static str, String> = self.meta.borrow().into();
+        let mut headers: HashMap<&'static str, String> = self.meta.borrow().into();
 
-        let mut req_builder = self
-            .endpoint
-            .to_request_builder(concat!("Libdatadog/", env!("CARGO_PKG_VERSION")))?
-            .header(
-                hyper::header::CONTENT_TYPE,
-                ddcommon::header::APPLICATION_MSGPACK,
-            )
-            .method(hyper::Method::POST);
-
-        for (key, value) in &headers {
-            req_builder = req_builder.header(*key, value);
-        }
+        headers.insert(
+            hyper::header::CONTENT_TYPE.as_str(),
+            ddcommon::header::APPLICATION_MSGPACK_STR.to_string(),
+        );
 
-        let req = req_builder.body(hyper::Body::from(body))?;
+        let strategy = RetryStrategy::default();
 
-        let resp = self.client.request(req).await?;
+        let result = send_with_retry(&self.endpoint, body, &headers, &strategy, None).await;
```

Comment:
nitpick: you can skip declaring strategy above and just pass `&RetryStrategy:default()` in the function call.